game_api.txt

```diff
@@ -227,6 +227,12 @@ New node def property:
  * Called when fire attempts to remove a burning node.
  * `pos` Position of the burning node.
 
+ `on_ignite(pos, igniter)`
+
+  * Called when a Flint and steel tool is used on a node.
```

Comment:
You should also mention that this stops a flame from being spawned - the mod would need to place their own flame